profiles/base.yaml

```diff
@@ -167,6 +167,9 @@ safe_room_empath:
 
 # Spellcasting (and Khri) settings
 use_harness_when_arcana_locked: false
+crossing_training_force_cambrinth: true
+buff_spells_force_cambrinth: true
```

Comment:
I'm not sure if I like `buff` in the setting name... when I initially saw this, I assumed it was used by `buff.lic`. Maybe it's fine given the next setting has `waggle` in the name (though I really dislike the naming confusion between `buff` and `waggle`, it has nothing to do with this PR). Thoughts?

Reply:
Yup, I agree and I was like 'ok, I'll use waggle'. I'll figure it out.

Reply:
I made the change.